its/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-ws</artifactId>
-      <version>7.9</version>
+      <version>7.9.1</version>
```

Comment:
is there a difference between the 2 versions? I wouldn't update this w/o a need...

I know it might not related to the `minimal version` specified [in this retro](https://discuss.sonarsource.com/t/learn-from-failure-sonarpython-1-15-1-bugfix-release/3291), but the conclusion from that retro
> Therefore, as a rule of thumb, when upgrading to a new LTS version the minimal SQ version of a plugin, it should be upgraded to the “root” version of this LTS and not a patched version.
> As technical release was done but not published publicly, decision was made to do a bugfix to downgrade minimal requirement to 7.9 instead of 7.9.1.

leads me to suggest to keep this version to the "root" version unless a good reason

Having `7.9` for minimal version of SQ and using `7.9.1` for the WS version can lead to confusion

WDYT?

Reply:
Ok. Makes sense to avoid confusion for now, since we don't actually need to update version in the ITs. But it might be needed if/when SQ introduces new APIs that the analyzer wants to use

Reply:
in that case, the SQ minimum version will have to be updated as well OR the plugin will have to deal with the two different APIs (to support LTS)